### PR TITLE
#ANC-253 Upgraded Bankruptcy Common Library to .NET 6

### PR DESCRIPTION
**Pull request by mafzal-nomensa**
Some unit tests are being ignored as agreed with team, a tech debt ticket has been created to fix these unit tests later as these depend on fixes for ODS common library. https://inssdigital.atlassian.net/browse/ANC-326

### CODE REVIEW

File: INSS.ODS.Bankruptcy.API.Common/Models/AdjReference/AdjRegistration.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+using System.Threading.Tasks;
+using INSS.ODS.Bankruptcy.API.Common.Models.Interfaces;
+using INSS.ODS.Bankruptcy.API.Common.Models.Interfaces.AdjReference;
+
+namespace INSS.ODS.Bankruptcy.API.Common.Models
+{
+    [DataContract]
+    public class AdjRegistration : IAdjRegistration
+    {
+        //[Index("Ix_Unique_ADJ_Constraint", IsUnique = true)]
```
Comment (matt-innes-inss) [inline INSS.ODS.Bankruptcy.API.Common/Models/AdjReference/AdjRegistration.cs:17]: There are quite a few of these indexes commented out.  Why?  Is there a potential performance impact of this?

File: INSS.ODS.Bankruptcy.API.Common.UnitTests/Models.Tests/Application/HomeAddressTests.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+using System.Linq;
+using INSS.ODS.API.Common.Utilities.TestHelpers;
+using INSS.ODS.Bankruptcy.API.Common.Models.Validators;
+using INSS.ODS.Bankruptcy.API.Common.Resources.Shared;
+using INSS.ODS.Bankruptcy.API.Common.UnitTests.Constants;
+
+namespace INSS.ODS.Bankruptcy.API.Common.UnitTests
+{
+    [TestClass]
+    [Ignore(TestConstants.ReasonForIgnoringUnitTests)]
```
Comment (matt-innes-inss) [inline INSS.ODS.Bankruptcy.API.Common.UnitTests/Models.Tests/Application/HomeAddressTests.cs:12]: Can you make sure a list of all the  ignored tests are put into a technical debt JIRA ticket? Or send me a list and I'll do it. Either is fine

**Review by matt-innes-inss — CHANGES_REQUESTED** [review]
Explain to me the Index thing and send me a list of Ignored tests and I'll approve it :-)

File: INSS.ODS.Bankruptcy.API.Common/Models/AdjReference/AdjRegistration.cs (hunk 1 above)
Comment (mafzal-nomensa) [reply INSS.ODS.Bankruptcy.API.Common/Models/AdjReference/AdjRegistration.cs]: The attributes have been updated on models, and removed from interfaces as EF Core does not support index attribute on properties but classes

File: INSS.ODS.Bankruptcy.API.Common.UnitTests/Models.Tests/Application/HomeAddressTests.cs (hunk 2 above)
Comment (mafzal-nomensa) [reply INSS.ODS.Bankruptcy.API.Common.UnitTests/Models.Tests/Application/HomeAddressTests.cs:12]: A ticket has been created and all ignored tests have been tagged as 'Ignored' https://inssdigital.atlassian.net/browse/ANC-326

**Review by matt-innes-inss — APPROVED** [review]
LGTM

**Pull request merged**